cli/githubutil.py

```diff
@@ -19,6 +19,49 @@
 from util import ctx, not_empty, info, warning, verbose, CliHint
 from github import GithubWebHookSyncer, CONCOURSE_ID
 from github.util import GitHubHelper, _create_github_api_object
+from github.util import _create_team, _add_user_to_team, _add_all_repos_to_team
+
+
+def assign_github_team_to_repo(
+    github_cfg_name: str,
+    github_org_name: str,
+    auth_token: CliHint(help="Token from an org admin user. Token must have 'admin:org' scope"),
+    team_name: str='ci'
+):
+    '''
+    Assign team 'team_name' to all repositories in organization 'github_org_name' and
+    give the team admin rights on those repositories. The team will be created if it does not exist
+    and the technical github user (from github_cfg_name) will be assigned to the team.
+    The token of the technical user must have the privilege to create webhooks (scope admin:repo_hook)
```

Comment:
it might be worthwile repeating that the given auth_token must grant 'admin:org' privileges (as already noted in the cli-hint)

Reply:
Done